bin/travis

```diff
@@ -18,6 +33,8 @@ if [[ $TRAVIS_BRANCH = "master" || $TRAVIS_BRANCH = "release-"* ]] && [[ $TRAVIS
   SBT_COMMAND="$SBT_COMMAND ;publish"
   if [[ $SBT_COMMAND == *"makeSite"* ]]; then
     SBT_COMMAND="$SBT_COMMAND ;ghpagesPushSite"
+    SBT_GHPAGES_COMMIT_MESSAGE=$(gh_pages_commit_message)
+    export SBT_GHPAGES_COMMIT_MESSAGE
```

Comment:
Drat.  I'll have a merge to do before #971 lands.  Yours worked first, so you win. :)